### PR TITLE
[Term Entry] General SDLC: Waterfall

### LINKED ISSUE #1810: [Term Entry] General SDLC: Waterfall
### Reason/inspiration (optional)

The [Software Development Lifecycle](https://www.codecademy.com/resources/docs/general/software-development-life-cycle) Model to cover:

- Waterfall: The goal is to create a minimum viable product (MVP). Each phase in the SDLC must be completed before moving on to the next phase.

Please write a new entry and expand on the above introductory paragraph under:  content/general/concepts/software-development-life-cycle/terms/ about the Waterfall model.

To get started with the PR and entry file:

Under content/general/concepts/software-development-life-cycle/terms/, create a new folder called `waterfall`. Then create a new file called `waterfall.md` under content/general/concepts/software-development-life-cycle/terms/`waterfall`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by gracekishino**
### Description

Added new folder `waterfall` inside `content/general/concepts/software-development-life-cycle/terms/`.
Added new entry file `content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md` for General SDLC: Waterfall.
Added a new image for the General SDLC: Waterfall entry phase diagram `sdlc-waterfall.jpg`.

Closes #1810

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+Title: 'Waterfall'
+Description: 'Models a Software Development Life Cycle (SDLC) where the goal is to create a minimum viable product (MVP). Each phase in the SDLC must be completed before moving on to the next phase.'
```
Comment (SSwiniarski) [inline content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md:3]:
```suggestion
Description: 'Models a Software Development Life Cycle (SDLC) where the goal is to create a minimum viable product (MVP).'
```
Extra info doesn't need to be in the description.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@gracekishino A solid entry. I only had one comment.

File: content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md (hunk 1 above)
Comment (gracekishino) [reply content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md]: @SSwiniarski thanks for reviewing! I took out the extra info.

**Review by SSwiniarski — APPROVED** [review]
@gracekishino Looks good for a second review.

File: content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  - 'Web Development'
+Tags:
+  - 'Development'
+  - 'Models'
+CatalogContent:
+  - 'learn-how-to-code'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **Waterfall** Model Software Development Life Cycle (SDLC) aims to create a minimum viable product (MVP). Each phase in the SDLC must be completed before moving on to the next. It is suited to projects where features are easily defined from the start.
```
Comment (caupolicandiaz) [inline content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md:15]:
```suggestion
The **waterfall** model is a Software Development Life Cycle (SDLC) that aims to create a minimum viable product (MVP). Each phase in this SDLC must be completed before moving on to the next. It is suited to projects where features are easily defined from the start.
```

File: content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+Title: 'Waterfall'
+Description: 'Models a Software Development Life Cycle (SDLC) where the goal is to create a minimum viable product (MVP).'
```
Comment (caupolicandiaz) [inline content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md:3]:
```suggestion
Description: 'A model of software development where the goal is to create a minimum viable product.'
```

File: content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  - 'Development'
+  - 'Models'
+CatalogContent:
+  - 'learn-how-to-code'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **Waterfall** Model Software Development Life Cycle (SDLC) aims to create a minimum viable product (MVP). Each phase in the SDLC must be completed before moving on to the next. It is suited to projects where features are easily defined from the start.
+
+The Waterfall Model was the first SDLC methodology to be created and was first documented in 1970 by an American computer scientist called Dr. Winston W. Royce, though the term "Waterfall" wasn't used until later.
```
Comment (caupolicandiaz) [inline content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md:17]:
```suggestion
The waterfall model was the first SDLC methodology to be created and was first documented in 1970 by American computer scientist Dr. Winston W. Royce, though the term "waterfall" wasn't used until later.
```

File: content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+The diagram below shows how the workflow represents the natural flow of a waterfall, with the stages being completed in linear, sequential steps:
```
Comment (caupolicandiaz) [inline content/general/concepts/software-development-life-cycle/terms/waterfall/waterfall.md:19]:
```suggestion
The diagram below shows how the workflow mirrors the natural flow of a waterfall, with the stages being completed in linear, sequential steps:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@gracekishino I've completed my review. Please update the entry and tag me when you've finished. Thanks

**Comment by gracekishino** [comment]
@caupolicandiaz thanks so much for reviewing! I have committed the suggested changes.

**Review by caupolicandiaz — APPROVED** [review]
@gracekishino Looks good.

**Comment by SSwiniarski** [comment]
@gracekishino Congrats on the new entry! Live link is here: https://www.codecademy.com/resources/docs/general/software-development-life-cycle/waterfall

image

**Comment by gracekishino** [comment]
@SSwiniarski So sorry I had typo in the filename for the image, so it isn't showing. I created a pull request to fix it.

**Comment by gracekishino** [comment]
@SSwiniarski realised you already had opened a pull request for the image filename typo, thank you for fixing that.

**Pull request merged**